README.md

```diff
@@ -0,0 +1,100 @@
+Apache Hadoop Ozone
+===
+
+Ozone is a scalable, redundant, and distributed object store for Hadoop. Apart from scaling to billions of objects of varying sizes, Ozone can function effectively in containerized environments such as Kubernetes and YARN.
+
+
+ * MULTI-PROTOCOL SUPPORT: Ozone supports different protocols like S3 and Hadoop File System APIs.
+ * SCALABLE: Ozone is designed to scale to tens of billions of files and blocks and, in the future, even more.
+ * CONSISTENT: Ozone is a strongly consistent object store. This consistency is achieved by using protocols like RAFT.
+ * CLOUD-NATIVE: Ozone is designed to work well in containerized environments like YARN and Kubernetes.
+ * SECURE: Ozone integrates with Kerberos infrastructure for access control and supports TDE and on-wire encryption.
+ * HIGHLY AVAILABLE: Ozone is a fully replicated system that is designed to survive multiple failures.
+
+## Documentation
+
+The latest documentation is generated together with the releases and hosted on the apache side.
+
+Please check [the documentation page](https://hadoop.apache.org/ozone/docs/) for more information.
+
+## Contact
+
+Ozone is part of the [Apache Hadoop](https://hadoop.apache.org) project.
+
+ * Ozone [web page](https://hadoop.apache.org/ozone/) is available from the Hadoop site
+ * Mailing lists
+     * For dev questions use: [hdfs-dev@hadoop.apache.org](https://lists.apache.org/list.html?hdfs-dev@hadoop.apache.org)
+     * For user questions use: [user@hadoop.apache.org](https://lists.apache.org/list.html?user@hadoop.apache.org)
+ * Chat: You can find the #ozone channel on the official ASF slack. Invite link is [here](http://s.apache.org/slack-invite).
+ * There are Open [Weekly calls](https://cwiki.apache.org/confluence/display/HADOOP/Ozone+Community+Calls) where you can ask anything about Ozone.
+     * Past meeting notes are also available from the wiki.
+
+
+## Download
+
+Latest release artifacts (source release and binary packages) are available from the Ozone web page:
+
+https://hadoop.apache.org/ozone/downloads/
+
+## Quick start
+
+### Run Ozone from published Docker image
+
+The easiest way to start a cluster with docker is:
+
+```
+docker run -P 9878 apache/ozone
```

Comment:
```suggestion
docker run -p 9878:9878 apache/ozone
```

I think `-p` is intended, and fixed port is needed for the `aws` commands below.  `-P` doesn't accept argument and would result in:

```
Unable to find image '9878:latest' locally
```